evergreen/config.yml

```diff
@@ -73,7 +73,7 @@ functions:
           export CMAKE_VARS_DIR=$(./evergreen/abspath.sh cmake_vars)
           source evergreen/cmake_vars_utils.sh
           if [ -n "${cmake_bindir|}" ]; then
-            export CMAKE=$(./evergreen/abspath.sh ${cmake_bindir}/cmake)
+            export CMAKE="$(./evergreen/abspath.sh "${cmake_bindir}/cmake")"
```

Comment:
I would have expected that the nested double quotes need to be escaped... but it appears to be working correctly?

Reply:
Yup. I couldn't find exactly where this behavior is documented in the bash manual, but each `$()` command substitution has its own context for quotes. It's also valid to nest them like `export CMAKE="$(./evergreen/abspath.sh "$(which "cmake")")"`